index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href="styles.css" rel="stylesheet">
+    <title>Portfolio: Mobile Version Skeleton</title>
+</head>
+<body>
+    <section class="mobile-skeleton">
+        <header class="header">
+            <nav class="navbar">
+                <ul class="header-container">
+                    <li class="logo"><h4>My Logo</h4></li>
+                    <li><img src="images/menu.png" alt="menu"></li>
+                </ul>
+            </nav>
+        </header>
+        <section class="headline">
+            <img src="images/headline.png" alt="headline">
+            <div class="main">
+                <p class="intro">I'm Kadenyi Glad to see you!</p>
+                <p class="pitch">I'm a software developer! I can help you build a product, feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+            </div>
+            <div class="connect">
+                <p>LET'S CONNECT</p>
+            </div>
+        </section>
+        <section class="contacts">
+                <img class="first" src="images/tweet.svg" alt="twitter">
+                <img src="images/the-i.svg" class="linked1" alt="letter_i">
+                <img src="images/the-n.svg" class="linked2" alt="letter_n">
+                <img class="first" src="images/the-m.svg" alt="letter_m">
+                <img class="first" src="images/cartoon.svg" alt="cartoon">
+                <img class="first" src="images/peace.svg" alt="peace">
```

Comment:
- [ ] please ensure that your social media icons are inside a `ul tag` and each of them inside a `li tag`.
- [ ] Add link place holders to each icon ..`<a href="/"></a>` .you can check how to add that [here](https://www.w3schools.com/html/html_links.asp)